### PR TITLE
Add material theme support

### PR DESCRIPTION
**Pull request by alexberazouski**
Fix lumo issue #60
Fix lumo issue when custom and default buttons look distorted

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-confirm-dialog/63)

### CODE REVIEW

**Comment by alexberazouski** [comment]
@jouni In the following case `cancel` is a default button and the rest are custom:

screenshot 2018-10-23 at 9 50 19

**Comment by alexberazouski** [comment]
https://github.com/vaadin/vaadin-confirm-dialog/issues/66

File: theme/material/vaadin-confirm-dialog-styles.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        width: 25em;
+        max-width: 100%;
+        margin-right: 24px;
+      }
+
+      [part="footer"] {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: flex-end;
+        margin-top: 28px;
```
[the remark this answers is not in the document]
Comment (jouni) [reply theme/material/vaadin-confirm-dialog-styles.html]: Actually, I looked at the specs wrong. I think 28px is fine after all.

**Comment by jouni** [comment]
> @jouni In the following case cancel is a default button and the rest are custom

Yeah, that case is tricky with MD, because they don’t really have that in the specs.

**Pull request merged**